src/applications/vaos/components/FacilityAddress.jsx

```diff
@@ -7,6 +7,8 @@ export default function FacilityAddress({
   name,
   facility,
   showDirectionsLink,
+  isV2,
```

Comment:
I think Narin switched these props to isHomepageRefresh elsewhere, which seems more clear to me.

Reply:
Fixed.